tools/java/org/unicode/cldr/util/CLDRFile.java

```diff
@@ -189,8 +189,10 @@ public boolean isNonInheriting() {
     public CLDRFile(XMLSource dataSource) {
         this.dataSource = dataSource;
         // source.xpath_value = isSupplemental ? new TreeMap() : new TreeMap(ldmlComparator);
-        creationTime = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'").format(Calendar.getInstance().getTime());
-        System.out.println("📂 Created new CLDRFile(dataSource) at " + creationTime);
+        if (DEBUG) {
```

Comment:
note that this is conflicts with #3

Reply:
actually, it might be better to take this out of this PR, since it's not related to Muscogee.

Reply:
Took it out per suggestion.